packages/ref-core/src/cmip_ref_core/pycmec/output.py

```diff
@@ -0,0 +1,55 @@
+from enum import StrEnum
+from typing import Optional
+
+from pydantic import (
+    BaseModel,
+)
+from pydantic.json_schema import GenerateJsonSchema, JsonSchemaValue
+
+
+class OutputCV(StrEnum):
+    """CMEC output bundle controlled vocabulary"""
+
+    INDEX = "index"
+    PROVENANCE = "provenance"
+    DATA = "data"
+    PLOTS = "plots"
+    HTML = "html"
+    METRICS = "metrics"
+    FILENAME = "filename"
+    LONG_NAME = "long_name"
+    DESCRIPTION = "description"
+
+
+class OutputIndex(BaseModel):
+    """CMEC output bundle index object"""
+
+    index: str = "index.html"
+
+
+class OutputProvenance(BaseModel):
+    """CMEC output bundle provenance object"""
+
+    environment: dict  # Key/value pairs listing all relevant diagnostic and framework environment variables.
+    modeldata: list[str, ...]  # type: ignore # Path to the model data used in this analysis.
```

Comment:
`list[str]` specifies that all items in the list are `str`s. `tuple[str, ...]` is the equiv for tuples.

```suggestion
    modeldata: list[str]  # type: ignore
    """Path to the model data used in this analysis."""
```

Reply:
will fix it.